libs/clients/rsk/personal-tax-return/src/lib/personalTaxReturn.config.ts

```diff
@@ -0,0 +1,26 @@
+import { defineConfig } from '@island.is/nest/config'
+import * as z from 'zod'
+
+const schema = z.object({
+  agentNationalId: z.string(),
```

Comment:
agent national id, is that the applicant?

Reply:
Nopes it is bascially user and pass for the api